src/main/java/ar/edu/unq/sarmiento/wicket/inscripcion/CargarInscripcionPage.html

```diff
@@ -0,0 +1,146 @@
+<wicket:extend>
+	<div class="row">
+		<div class="panel panel-primary">
+			<div class="panel-heading titulo">
+				<h4>Inscribir alumno</h4>
+			</div>
+			<div class="panel-body">
+				<form wicket:id ="inscripcionAlumno" class="form-horizontal">
+				
+					<div class="form-group">
+						<label for="nombre" class="col-md-2 control-label"> Nombre completo</label>
+						<div class="col-md-9">
+							<input wicket:id ="nombre" type="text" class="form-control"
+									id="nombre"   placeholder="Ileana" >
+						</div>
+					</div>
+					<div class="form-group">
+					<label for="genero" class="col-md-2 control-label">Género </label>
+					<div class="col-md-9">
+						<input type="text" class="form-control" id="genero"
+							wicket:id="genero">
+					</div>
+				</div>
+				<div class="form-group">
+					<label for="dni" class="col-md-2 control-label">DNI </label>
+					<div class="col-md-9">
+						<input type="text" class="form-control" id="dni"
+							wicket:id="dni">
+					</div>
+				</div>
+				<div class="form-group">
+					<label for="fechaNacimiento" class="col-md-2 control-label">Fecha de nacimiento </label>
+					<div class="col-md-9">
+						<input type="text" class="form-control" id="fechaNacimiento"
+							wicket:id="fechaNacimiento">
```

Comment:
Estaría bueno ponerle un placeholder a este:
```suggestion
							wicket:id="fechaNacimiento" placeholder="30/10/1991">
```